pkg/tbtc/maintainer/local_bitcoin_difficulty_chain_test.go

```diff
@@ -0,0 +1,130 @@
+package maintainer
```

Comment:
Maybe just `chain_test.go`? This will be shorter and will indicate that this is a test implementation of interfaces placed in `chain.go` (we have the same pattern in `pkg/tbtc`).

Reply:
Done in 70604b7878805b61aa7b60325d603d656c3799c8.